public/js/components/AtomEdit/CustomEditors/StoryQuestionFields/Question.js

```diff
@@ -19,29 +23,93 @@ export class StoryQuestionsQuestion extends React.Component {
+    super(props);
+    this.state = {
+      answers: props.fieldValue && props.fieldValue.answers && props.fieldValue.answers.length ?
+        props.fieldValue.answers :
+        []
+    };
+  }
+
+  componentDidMount() {
+    this.state.answers.forEach((answer, i) => {
+    // eslint-disable-next-line
+    const [_, atomType, atomId] = answer.answerId.split('/');
+      fetchCapiAtom(atomType, atomId)
+        .then(atom => {
+          this.setState(prevState => {
+            const answers = prevState.answers.slice();
+            answers[i].title = atom.title;
+          });
+        });
+    });
+  }
+
   updateQuestion = (questionObject) => {
     const questionWithId = Object.assign({}, questionObject, {
-      questionId: questionObject.questionId || uuidv4()
+      questionId: questionObject.questionId || uuidv4(),
+      answers: this.state.answers
     });
 
     this.props.onUpdateField(questionWithId);
   }
 
-  render () {
-    const answersCount = (this.props.fieldValue && this.props.fieldValue.answers) ? this.props.fieldValue.answers.length : 0;
+  onSelect = (snippet) => {
+    this.setState(
+      (oldState) => Object.assign({}, oldState, { answers: oldState.answers.concat({
+        title: snippet.title,
+        answerId: `atom/${snippet.atomType}/${snippet.id}`, 
+        answerType: 'ATOM' 
+      })}),
+      () => {
+        this.props.onUpdateField({
+          questionId: this.props.fieldValue.questionId,
+          questionText: this.props.fieldValue.questionText,
+          answers: this.state.answers
+        });
+      }
+    );
+  }
+  
+  onDelete = (i) => {
+    this.setState(
+      (oldState) => Object.assign({}, oldState, {
+        answers: oldState.answers.slice(0, i).concat(oldState.answers.slice(i + 1))
+      }),
+      () => {
+        this.props.onUpdateField({
+          questionId: this.props.fieldValue.questionId,
+          questionText: this.props.fieldValue.questionText,
+          answers: this.state.answers
+        });
+      }
+    );
+  }
 
+  render () {
     return (
       <div className="form__field form__field--nested">
         <ManagedForm data={this.props.fieldValue} updateData={this.updateQuestion} onFormErrorsUpdate={this.props.onFormErrorsUpdate} formName="storyquestionsEditor">
           <ManagedField fieldLocation="questionText" name="Question" isRequired={true}>
             <FormFieldTextInput />
           </ManagedField>
-          <ManagedField fieldLocation="answers" name={`Answers (${answersCount})`} >
-            <FormFieldArrayWrapper>
-              <StoryQuestionsAnswer/>
-            </FormFieldArrayWrapper>
-          </ManagedField>
         </ManagedForm>
+        <div className="atom__answers__header">
+          <h5>Answers</h5>
+          <SearchSuggestions fieldPlaceholder="Search for snippets" filters={filters} onSelect={this.onSelect} />
+        </div>
+        <ul className="atom__answers">
+          {this.state.answers.map((answer, i) => 
```

Comment:
The only way I can see this could reuse other components is to use the `FormFieldArrayWrapper` but it's a bit specialised, I think it'd be too much effort.

Reply:
yeah no, let's not do that 😅 I'm happy to refactor that module (the array wrapper one) another time, there are some abstraction leaks which make it impractical to work with